docs/developer-docs/smart-contracts/best-practices/troubleshooting.mdx

```diff
@@ -14,6 +14,20 @@ When a dapp or canister is running with a high amount of latency, there are some
 
 This document will detail some troubleshooting methods and performance optimizations that developers can use when troubleshooting latency.
 
+## Troubleshooting subnet latency
+
+On subnets with low loads, query calls are answered in about 100 milliseconds and update calls are answered in about 2 seconds.
+
+On subnets with higher loads, your application may experience elevated latencies or reduced accessibility. To get more information about the subnet your canister is running on, view the canister's information on the [ICP dashboard](https://dashboard.internetcomputer.org/canisters).
+
+### Recommendation: Consider moving the canister to another subnet.
```

Comment:
I don't think we can really recommend this in the docs yet, because there are caveats to it, so I'd rather keep this section out.

Reply:
we could instead point users to compute allocations

Reply:
Revised to recommend using compute allocations instead of moving canisters.